test/main.cpp

```diff
@@ -27,8 +27,8 @@ int main(int argc, char *argv[]) {
 
   // This comment is a reminder for whenever we can test using a Mac
   // machine.
-  //   format.setMajorVersion(4);
-  //   format.setMinorVersion(1);
+  format.setMajorVersion(3);
+  format.setMinorVersion(2);
```

Comment:
is this required?

Reply:
Conclusion: yes it's required for the tests on CI as the default opengl version used by qt is 2.0 - but it's not needed elsewhere as we do

`  QSurfaceFormat::setDefaultFormat(QVTKOpenGLNativeWidget::defaultFormat());`

in the GUI, and using this line instead of setting the version in the tests also works fine in CI:

https://github.com/spatial-model-editor/spatial-model-editor/actions/runs/8247110130/job/22554572540